.github/workflows/php-compatibility.yml

```diff
@@ -0,0 +1,31 @@
+name: PHP Compatibility
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
@faisal-alvi Just a heads up that we needed to change this to `master` since there isn't a `trunk` branch in this repo